src/api/dataFilter.tsx

```diff
@@ -0,0 +1,13 @@
+import { IData } from "../interfaces/IData";
+
+class dataFilter {
```

Comment:
DataFilter isn't best name since data can be anything. And also consider that if you add some more methods e.g. about sorting, will you rename it to dataSort? Please rename this one, be more specific for understandability